client/app/telecom/pack/xdsl/modem/dmz/pack-xdsl-modem-dmz.html

```diff
@@ -61,7 +60,7 @@ <h2 class="widget-presentation-title" data-translate="xdsl_modem_dmz_status"></h
                                     title="{{DmzCtrl.modemLan.IPAddress}} {{ 'common_table_edit' | translate }}"
                                     data-ng-click="DmzCtrl.setEditMode()"
                                     data-ng-disabled="!DmzCtrl.mediator.capabilities.canChangeDMZ || !DmzCtrl.modemLan || DmzCtrl.mediator.tasks.changeModemConfigDMZ">
-                                <span class="ovh-font ovh-font-editer fs16" aria-hidden="true"></span>
+                                <span class="ovh-font ovh-font-editer h6" aria-hidden="true"></span>
```

Comment:
Pay attention, using class like `hX` with icons from `ovh-font` it may cause fonts weight issues!

Reply:
Nvm, this one is useless here.